lib/resources/pg_hba_conf.rb

```diff
@@ -0,0 +1,107 @@
+# encoding: utf-8
+# copyright: 2017
+# author: Rony Xavier,rx294@nyu.edu
+# author: Aaron Lippold, lippold@gmail.com
+# license: All rights reserved
+
+require 'resources/postgres'
+
+module Inspec::Resources
+  class PGHbaConf < Inspec.resource(1)
+    name 'pg_hba_conf'
+    desc 'Use the `pg_hba_conf` InSpec audit resource to test the client
+          authentication data defined in the pg_hba.conf file.'
+    example "
+      describe pg_hba_conf.where { type == 'local' } do
+        its('auth_method') { should eq ['peer'] }
+      end
+    "
+
+    attr_reader :conf_dir, :conf_file, :params
+
+    # @todo add checks to ensure that we have data in our file
+    def initialize(hba_conf_path = nil)
+      if inspec.os.linux?
```

Comment:
Hi @rx294 I haven't seen an update to this PR but you mentioned you made corrections to both. Just wanted you to be aware in case you forgot to push up your changes. Thanks! I also left another change request on #1963 to help clean that up to get it ready. You'll likely have to make a similar change to this one in order to mock/stub the calls to the `postgres` resource.